src/pages/Profile.jsx

```diff
@@ -1,16 +1,19 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
 import 'antd/dist/antd.css';
 import { Tabs } from 'antd';
+import { AFCBackend } from '../util/utils';
 import ProfileGeneralInfo from '../components/profile/ProfileGeneralInfo';
 import ProfileDUICrimHistory from '../components/profile/ProfileDUICrimHistory';
 import ProfileRolesSkills from '../components/profile/ProfileRolesSkills';
 import ProfileAvailability from '../components/profile/ProfileAvailability';
 import WaiversGrid from '../components/waivers/WaiversGrid';
-import VolunteeringHistory from '../components/profile/VolunteeringHistory';
+// import VolunteeringHistory from '../components/profile/VolunteeringHistory';
 import '../components/profile/profile.css';
 
 const { TabPane } = Tabs;
 function Profile() {
+  const { userId } = useParams();
```

Comment:
Let's add a todo here to redirect the user to their own profile (set user id = their own id) if they aren't an admin or if the user id isn't provided